website/docs/r/cloud_provider_snapshot_backup_policy.html.markdown

```diff
@@ -72,6 +75,197 @@ resource "mongodbatlas_cloud_provider_snapshot_backup_policy" "test" {
+
+```hcl
+resource "mongodbatlas_cluster" "my_cluster" {
+  project_id   = "<PROJECT-ID>"
+  name         = "clusterTest"
+  disk_size_gb = 5
+
+  //Provider Settings "block"
+  provider_name               = "AWS"
+  provider_region_name        = "EU_CENTRAL_1"
+  provider_instance_size_name = "M10"
+  provider_backup_enabled     = true // must be enabled in order to use cloud_provider_snapshot_backup_policy resource
+  provider_disk_iops          = 100
+  provider_encrypt_ebs_volume = false
+}
+
+resource "mongodbatlas_cloud_provider_snapshot_backup_policy" "test" {
+  project_id   = mongodbatlas_cluster.my_cluster.project_id
+  cluster_name = mongodbatlas_cluster.my_cluster.name
+
+  reference_hour_of_day    = 3
+  reference_minute_of_hour = 45
+  restore_window_days      = 4
+
+
+  policies {
+    id = mongodbatlas_cluster.my_cluster.snapshot_backup_policy.0.policies.0.id
+
+    # Item removed
+    # policy_item {
+    #   id                 = mongodbatlas_cluster.my_cluster.snapshot_backup_policy.0.policies.0.policy_item.0.id
+    #   frequency_interval = 1
+    #   frequency_type     = "hourly"
+    #   retention_unit     = "days"
+    #   retention_value    = 1
+    # }
+
+    # Item removed
+    # policy_item {
+    #   id                 = mongodbatlas_cluster.my_cluster.snapshot_backup_policy.0.policies.0.policy_item.1.id
+    #   frequency_interval = 1
+    #   frequency_type     = "daily"
+    #   retention_unit     = "days"
+    #   retention_value    = 2
+    # }
+
+    # Item removed
+    # policy_item {
+    #   id                 = mongodbatlas_cluster.my_cluster.snapshot_backup_policy.0.policies.0.policy_item.2.id
+    #   frequency_interval = 4
+    #   frequency_type     = "weekly"
+    #   retention_unit     = "weeks"
+    #   retention_value    = 3
+    # }
+
+    policy_item {
+      id                 = 5f0747cad187d8609a72f546
+      frequency_interval = 5
+      frequency_type     = "monthly"
+      retention_unit     = "months"
+      retention_value    = 4
+    }
+  }
+}
+```
+
+-> **NOTE:** (See text in first example for more details on the default.) If you want the Cloud Backup Snapshot Policy to vary in number of policies for a cluster that was already created/imported, perhaps you want to remove three policy items and modify the remaining policy, simply follow the above example here.
+
+~> **IMPORTANT:** Note in this example we decided to remove the first 3 items so we can't use `mongodbatlas_cluster.my_cluster.snapshot_backup_policy.0.policies.0.policy_item.3.id` this sentence to retrieve the monthly id value of the cluster state due to once the cluster being modified or makes a `terraform refresh` will cause that the three items will remove from the state, so we will get an error due to the index 3 doesn't exists any more and our monthly policy item is moved to the first place of the array.(See text in the first example for more details on it)
```

Comment:
this should be under the heading, Example Usage - Remove 3 Default Policies Items After the Cluster Has Already Been Created, as it is explaining what the example is.

The IMPORTANT probably is best to put near 256 in a comment or use the other explanation cut and pasted in here.